### ISSUE TITLE
Add OPUS support for cassini_uvis_solarocc_beckerjarmak2023 bundle

### ISSUE THREAD

**Issue by rfrenchseti** [SETI/rms-pdsfile#59]
Add OPUS support for the `cassini_uvis_solarocc_beckerjarmak2023` PDS4 bundle: `opus_products`, `opus_types`, etc.

**Comment by matthewtiscareno** [comment]
Can this be done in parallel with [rms-data-projects #117](https://github.com/SETI/rms-data-projects/issues/117)?

**Comment by rfrenchseti** [comment]
Yes, they are essentially unrelated tasks. We just can't do the final testing until 117 is finished.

**Comment by rfrenchseti** [comment]
- Filenames look like `cassini_uvis_solarocc_beckerjarmak2023/data/uvis_euv_2005_159_solar_time_series_ingress.xml/tab` .
- The OPUS ID will be `co-uvis-occ-2005-159-sun-i` where the 2005, 159, and i are taken from the filename.
- This primary xml/tab file will have the opus type `('Cassini UVIS Solar Occultations', 10, 'couvis_solar_occ_ring', 'Occultation Ring Time Series', True)`
- For `uvis_euv_2008_083_solar_time_series_egress.tab` only there are also files `cassini_uvis_solarocc_beckerjarmak2023/data/supplemental/uvis_euv_2008_083_solar_time_series_egress_supplement.xml/tab` which will have the opus type (`'Cassini UVIS Solar Occultations', 20, 'couvis_solar_occ_ring_supp', 'Occultation Ring Time Series Supplemental', True)`
- For all files include the four xml/pdf files under `cassini_uvis_solarocc_beckerjarmak2023/document/` with the type `('Cassini UVIS Solar Occultations', 30, 'couvis_solar_occ_documentation', 'Documentation', False)`
- For all files include the associated xml/jpg files under `cassini_uvis_solarocc_beckerjarmak2023/browse/` with the type `('Cassini UVIS Solar Occultations40, 'couvis_solar_occ_browse', 'Detailed Browse', True)`
- For the rules for preview images and diagrams, you can look at the structure here: https://staging.pds.seti.org/pds4/previews/cassini_uvis_solarocc_beckerjarmak2023/
